### PR TITLE
Add deprecation warnings in the docs for commands being removed in 0.19.0

### PR DESCRIPTION
**Pull request by jmholzer**
Signed-off-by: Jannic Holzer <jannic.holzer@quantumblack.com>

## Description
In Kedro version 0.19.0, six commands are being deprecated (#1617). This PR adds deprecation notices for these commands to the documentation.

## Development notes

I manually checked that the docs still build successfully.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1912"></a>

### CODE REVIEW

**Review by AhdraMeraliQB — APPROVED** [review]
Good catch on this!

File: docs/source/development/commands_reference.md
Comment (MerelTheisenQB) [inline docs/source/development/commands_reference.md:284]: We don't really use this style of formatting anywhere else. Perhaps just add this as a {note} or with `>` indenting. 

See https://github.com/kedro-org/kedro/blob/main/docs/source/tutorial/create_pipelines.md for example of "notes"

File: docs/source/development/commands_reference.md
Comment (MerelTheisenQB) [inline docs/source/development/commands_reference.md:405]: For the `kedro test` and `kedro lint` commands we can link to the new doc pages.

**Review by noklam — COMMENTED** [review]
Looks good, not much too added. Agree with Merel we should stick with the {note} style.

File: docs/source/development/commands_reference.md
Comment (jmholzer) [reply docs/source/development/commands_reference.md:405]: I agree this is a good idea, though I would prefer to wait until #1904 is merged, if that sounds OK.

File: docs/source/development/commands_reference.md
Comment (jmholzer) [reply docs/source/development/commands_reference.md]: Fixed in the latest commits.

File: docs/source/development/commands_reference.md
Comment (jmholzer) [reply docs/source/development/commands_reference.md]: I understand, I fixed it to use note syntax in the latest commits.

**Comment by jmholzer** [comment]
Thanks for the reviews @noklam and @MerelTheisenQB!

I made the suggested changes.

File: docs/source/development/commands_reference.md
Comment (MerelTheisenQB) [reply docs/source/development/commands_reference.md]: I didn't mean to remove the "(deprecated from version 0.19.0)" completely, just to have it outside of the linked text so it would be: [`kedro activate-nbstripout`](#strip-output-cells)(deprecated from version 0.19.0) etc.

Comment (jmholzer) [reply docs/source/development/commands_reference.md]: Sure, I fixed this in the latest commit.

**Review by MerelTheisenQB — APPROVED** [review]
LGTM! 😄

**Pull request merged**
